src/cloud/components/Blocks/BlockToolbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropsWithChildren } from 'react'
+import Portal from '../../../design/components/atoms/Portal'
+import Toolbar, {
+  ToolbarControlProps,
+} from '../../../design/components/organisms/Toolbar'
+
+const BlockToolbar = ({
+  controls = [],
+  children,
+}: PropsWithChildren<{
+  controls?: ToolbarControlProps[]
+}>) => {
+  const portalContainer = document.getElementById(
+    'block__editor__view__toolbar-portal'
+  )
+
+  if (portalContainer == null) {
+    return null
+  }
+
+  return (
+    <Portal target={portalContainer}>
```

Comment:
I find it easier personally to write `Portal` and get the import rather than having to write `reactDom.createPortal` and import `react-dom` everytime. 

I also want to extend the Portal Component to introduce the null check for the target container